app/models.py

```diff
@@ -2364,3 +2384,18 @@ def serialize(self):
             "updated_at": get_dt_string_or_none(self.updated_at),
             "logged_in_at": get_dt_string_or_none(self.logged_in_at),
         }
+
+
+class LetterAttachment(db.Model):
+    __tablename__ = "letter_attachment"
+
+    id = db.Column(UUID(as_uuid=True), primary_key=True, nullable=False, default=uuid.uuid4)
+
+    created_at = db.Column(db.DateTime, nullable=True, default=datetime.datetime.utcnow)
```

Comment:
Maybe nullable should be False here? Since each attachment will have it?

Reply:
Oh my, good spot